.github/workflows/deploy.yml

```diff
@@ -74,6 +74,7 @@ jobs:
       - name: Copy built schema to SCHEMA_DIR
         run: | 
           cp -r dist/schemata/ $SCHEMA_DIR/
+          rm -r $SCHEMA_DIR/dev
```

Comment:
```suggestion
          rm -r $SCHEMA_DIR/dev
          cp -r dist/schemata/ $SCHEMA_DIR/
```

Reply:
Switch positions. (First remove dev folder in the schema dir, then copy the new content over)